versions.txt

```diff
@@ -0,0 +1,4 @@
+# Format:
+# module:released-version:current-version
+
+cloud-sql-java-connector:1.0.16:1.0.17-SNAPSHOT
```

Comment:
So there is always a versions.txt? 

Does (or can) this update the version numbers in the file for the useragent as well?

Reply:
yeah. since we're reading the version for the user agent in from a pom.properties file

Reply:
From what I understand, the `versions.txt` is the source of truth when it comes to the package versioning, and we can add comments in the pom files to sync up those versions with the version in `versions.txt`

Reply:
SGTM